sdks/python/apache_beam/coders/coders.py

```diff
@@ -572,6 +572,16 @@ def _create_impl(self):
   def to_type_hint(self):
     return typehints.Optional[self._value_coder.to_type_hint()]
 
+  def _get_component_coders(self):
+    # type: () -> List[Coder]
+    return [self._value_coder]
+
+  @classmethod
+  def from_type_hint(cls, typehint, registry):
+    value_type = list(
```

Comment:
You just expect one inner type here as well ?
If so we should probably validate that and simplify this code.